src/snitch_reporter_teamcity.cpp

```diff
@@ -148,6 +148,8 @@ void report(const registry& r, const snitch::event::data& event) noexcept {
                 send_message(r, "testFinished", {{"name", make_full_name(e.id)}});
 #    endif
             },
+            [&](const snitch::event::section_started&) {},
+            [&](const snitch::event::section_ended&) {},
```

Comment:
TeamCity uses the following:
 - `##teamcity[blockOpened name='...' description='...']`
 - `##teamcity[blockClosed name='...']`

Reply:
In the  `print_assertion()` is see where it iterates over the sections, but I don't see how it outputs the test you show here. I don't see and "blockOpened" or "blockCosed"?

Reply:
Got it :)